### DISCUSSION TITLE
[零协会网站]“切换节点”“切换API节点”问号直达网页缺失

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [LocalizeLimbusCompany/LocalizeLimbusCompany#297] [问答 | Q&A] [answered]
如下图所示 工具箱为从github获得的v1.0.2
[Log-20241026.log](https://github.com/user-attachments/files/17531016/Log-20241026.log)
![image](https://github.com/user-attachments/assets/cd94fcfd-6147-4f88-99c0-1c9303a47341)
![image](https://github.com/user-attachments/assets/2e5a91cd-3796-4c0d-a24c-4fbccfecd109)

**Comment by Nephthys137** [comment]
@ZengXiaoPi

**Comment by ZengXiaoPi** [comment, marked as answer]
下个版本修复。
并且这种问题应当去工具箱的Issue提问。
